From d347deb0376a3254d7cd8bbfe46dd645d1bf1061 Mon Sep 17 00:00:00 2001
From: "Yann E. MORIN" <yann.morin.1998@free.fr>
Date: Fri, 1 Dec 2017 11:08:16 +0100
Subject: [PATCH] install: samples need the data files

When installing samples, "sample voicemail" is generated from the
already-installed sound files.

However, when doing the install and the samples at the same time in a
parallel install, it is possible that the sound files are not already
installed at the time we try to generate the voicemail data.

Ensure the needed dependency.

Signed-off-by: Julien Olivain <ju.o@free.fr>
[Julien: rebase patch on tag 20.7.0]
Signed-off-by: "Yann E. MORIN" <yann.morin.1998@free.fr>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 7ef73342cc..bb60f5e380 100644
--- a/Makefile
+++ b/Makefile
@@ -819,7 +819,7 @@ adsi:
 		$(INSTALL) -m 644 "$$x" "$(DESTDIR)$(ASTETCDIR)/`$(BASENAME) $$x`" ; \
 	done
 
-samples: adsi
+samples: adsi datafiles
 	@echo Installing other config files...
 	$(call INSTALL_CONFIGS,configs/samples,.sample)
 	$(INSTALL) -d "$(DESTDIR)$(ASTSPOOLDIR)/voicemail/default/1234/INBOX"
-- 
2.45.2

